master/buildbot/steps/master.py

```diff
@@ -38,18 +38,18 @@ class MasterShellCommand(BuildStep):
     description = 'Running'
     descriptionDone = 'Ran'
     descriptionSuffix = None
-    renderables = ['command', 'env', 'workdir']
+    renderables = BuildStep.renderables + ['command', 'env']
```

Comment:
I need to check whether it really accumulates them automatically.

In any case, a seemingly simple change caused a test case to error.